pwiz_tools/Skyline/Executables/BuildMethod/BuildWatersMethod/BuildWatersMethod.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>BuildWatersMethod</RootNamespace>
     <AssemblyName>BuildWatersMethod</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>

```

Comment:
Presumably intentional?

Reply:
Yes, the dll Waters provided is built on 3.5, so I have to use the same framework for the executable to ensure compatibility.

Reply:
I expected as much, but sometime Visual Studio gets ideas....